pkg/parent/parent.go

```diff
@@ -153,6 +149,12 @@ func Parent(opt Opt) error {
 		}()
 	}
 
+	// after child is fully configured, write PID to child_pid file
+	childPIDPath := filepath.Join(opt.StateDir, StateFileChildPID)
```

Comment:
thanks, but this should be after `portDriverInitComplete`

Also, probably the parent should remove the file before starting the child if the file already exists

Reply:
This one is not enough https://github.com/rootless-containers/rootlesskit/blob/master/pkg/parent/parent.go#L73 ?